src/store/modules/accounts/hdWallet.js

```diff
@@ -94,31 +94,31 @@ export default {
       txBase64,
       opt: { modal = true, host = null },
     }) {
-      const sdk = rootGetters['sdkPlugin/sdk'];
+      const networkId = await (rootGetters['sdkPlugin/sdk'] ?? rootGetters['sdk13Plugin/sdk'].api).getNetworkId();
```

Comment:
My draft PR actually removes the SDK Vuex module so I strongly suggest to use the composable. Please see how it's done here: https://github.com/aeternity/superhero-wallet/pull/1731/files#diff-37085670e4cc4d0853535bb598e3c36b31ffb49af1badbebe2018c1591d51b9c

On the other hand side in one of my recent PRs Nikita suggested to not use the `sdk.getNetworkId()` but the getter `activeNetwork.networkId` which is more safe for us.

Reply:
I am using getters for choosing network Id, instead of `sdk.getNetworkId()`.
As I said, I am reducing refactoring changes with the SDK task as that is not the motive of the task.

Reply:
Using `activeNetwork` getter